### ISSUE TITLE
Chat - Links in headers are not formatted as links

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#31352]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.99-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause-Internal Team
**Slack conversation:** @

## Action Performed:
1. Go to staging.new.expensify.com or open Desktop App
2. Login with any account
3. Go to any chat
4. Send a message with a header that contains a link like - # Header [www.google.com](http://www.google.com/)

## Expected Result:
Link in the header is formatted as a link in the sent message

## Actual Result:
Link in headers are not formatted as link in the sent message

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/115492554/ca9d44c5-22c0-43e2-966e-c0c71772c9e3

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by eh2077** [comment]
This is a regression from https://github.com/Expensify/App/issues/26942 which is still on hold for regression.

**Comment by robertKozik** [comment]
Hey sorry for the delay. Regression has been already fixed and `expensify-common` was updated accordingly. I think we can close this issue out

<img width="634" alt="image" src="https://github.com/Expensify/App/assets/61146594/275c96ae-5bd7-40ec-a096-00111673739b">

**Comment by JmillsExpensify** [comment]
Didn't the proposal for that PR come much earlier? https://github.com/Expensify/App/issues/26942#issuecomment-1744062072

**Comment by JmillsExpensify** [comment]
@parasharrajat Mind chiming in on [this comment](https://github.com/Expensify/App/issues/31352#issuecomment-1825284856)? I don't have enough context to know if payment is due or not.

**Comment by JmillsExpensify** [comment]
Discussed internally and we agreed on 25% compensation. So payment summary:

 - Issue reporter: N/A
 - Contributor: @tienifr $125

**Comment by JmillsExpensify** [comment]
@parasharrajat I don't think you reviewed the PR in question, but let me know if I've misunderstood. 

@tienifr Send you an offer on Upwork.

**Comment by JmillsExpensify** [comment]
All paid out!
